docs/semgrep-appsec-platform/jira.md

```diff
@@ -121,6 +121,10 @@ Once a ticket has been created, a link appears on the right side of the **Findin
 
 ### Supply Chain
 
+:::info
+Only users who’ve worked with previous versions of the Jira integration can create SCA tickets. Semgrep is working on releasing an update that allows all users to create SCA tickets.
```

Comment:
Do I need to be more specific and say something like "Only users who’ve worked with previous versions of the Jira integration can create SCA tickets; support for new users of the Jira integration will be able to create SCA tickets with a forthcoming release" ?